Feature/lp52 elevated card with icon and text

Lupillo17 opened the pull request:
- Added functionality of a Card with icon and text only, also was added a example for usage.
- Added Typography to cards
- Merge changes from develop
- Delete misc.xml from VCS
- Added example module to easy test the app changes when the app runs

 Jira: https://devaptivist.atlassian.net/browse/LP-52?atlOrigin=eyJpIjoiZmExNGU4M2Y1MmY4NDE1OTlkNDY0MTA1NmNjNDM5MjQiLCJwIjoiaiJ9

Result
![image](https://user-images.githubusercontent.com/33376944/229172240-edf4e154-0239-45c4-8b0b-d046a1fcba59.png)

melissaquezada94 commented:
Hi Lupillo,
Would be nice that when you create the PR, you add the name of the task, the task's link to navigate to Jira and a brief overview of what was added, deleted or updated in the code. So it's easier to do the code review, please.

melissaquezada94 commented:
Another observation it's that you could do a squash in your commits to have just one commit per PR.

Lupillo17 commented:
@melissaquezada94 Hi Melissa I updated the description whit a image of final result, also I pot Jiras link from the related task.

- Added functionality of a Card with icon and text only, also was added a example for usage.
- Added Typography to cards
- Merge changes from develop
- Delete misc.xml from VCS

 Jira: https://devaptivist.atlassian.net/browse/LP-52?atlOrigin=eyJpIjoiZmExNGU4M2Y1MmY4NDE1OTlkNDY0MTA1NmNjNDM5MjQiLCJwIjoiaiJ9

Result
![image](https://user-images.githubusercontent.com/33376944/229172240-edf4e154-0239-45c4-8b0b-d046a1fcba59.png)
